src/ResourceManager/DataLakeStore/Commands.DataLakeStore/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.DataLake.Store" version="2.2.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DataLake.Store" version="2.3.0-preview" targetFramework="net452" />
```

Comment:
@ro-joowan same comment about removing the references in this file

Reply:
Done - thanks!